packages/block-library/src/term-description/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Server-side rendering of the `core/term-description` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/term-description` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_term_description( $attributes, $content, $block ) {
+
+	if ( ! is_category() && ! is_tag() && ! is_tax() ) {
+		return '';
+	}
+	$classes = 'term-description';
+	if ( isset( $attributes['textAlign'] ) ) {
+		$classes .= ' has-text-align-' . $attributes['textAlign'];
+	}
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
+
+	return '<div ' . $wrapper_attributes . '>' . term_description() . '</div>';
```

Comment:
I'm not 100% sure with this wrapper `div`. Do you think we can avoid it with `term_description` or somehow? Not strong opinions though, just seems extraneous.

Reply:
The term-description can contain anyhthing... the field accepts HTML, so it's vital to have a wrapper in order to properly apply styles to it.
I checked into using a filter and it's certainly possible... but it would add another layer of complexity without any real benefit, the end result would be the same, just expressed differently 👍